cscs-checks/prgenv/gpu_direct_acc.py

```diff
@@ -1,25 +1,22 @@
-import os
-
+import reframe as rfm
 import reframe.utility.sanity as sn
-from reframe.core.pipeline import RegressionTest
-
 
-class GpuDirectAccCheck(RegressionTest):
+@rfm.simple_test
+class GpuDirectAccCheck(rfm.RegressionTest):
     def __init__(self, **kwargs):
-        super().__init__('gpu_direct_acc_check',
-                         os.path.dirname(__file__), **kwargs)
+        super().__init__()
```

Comment:
1. Are we happy with the auto-generated name?
2. Does it make sense to provide a short human-readable description of the test?

Reply:
@ajocksch Can you please address/answer these? Other than that, this PR is ready to be merged.